paket-files/haf/YoLo/YoLo.fs

```diff
@@ -622,7 +622,7 @@ module App =
 
   /// Get the assembly resource
   let resource name =
-#if DNXCORE50
+#if NETSTANDARD1_5
```

Comment:
that's probably a good solution for the referenced `Yolo.fs` right now - @haf, do you agree? Do you want me also to revert other `#IF DNXCORE` ==> `#IF NETSTANDARD` changes in Suave proj, or should they use the new framework code?